### DISCUSSION TITLE
503 Service Temporarily Unavailable

### DISCUSSION THREAD

**Discussion by qlxdev** [kirimwa-id/forum#165] [Tanya Jawab]
Selamat malam mas rio,  mohon infonya sedari pagi memang masih trouble ya??🙏 Kok response yg di terima 503. Terimakasih
![IMG_20230209_203929](https://user-images.githubusercontent.com/64348777/217828431-90f0e4b1-0211-4314-a3d9-b050162616a8.jpg)

**Comment by ekoyuliawan** [comment]
Kejadian ini terus berulang sepertinya. Sudah 3x mengalami error yang sama dalam 3 hari terakhir.

**Reply by rioastamal** [reply]
Alasan utama kenapa akhir-akhir ini sering terjadi down ada di #166. Sebagai langkah untuk mengurangi downtime beberapa pengguna (free) dengan quota yang sangat tinggi akan dikurangi quotanya.

**Reply by dp20bp** [reply]
mas @rioastamal , dengan pilihan berbayar. berapa SLA yang di bisa kami dapatkan . utamanya tidak terjadi down-time

**Reply by rioastamal** [reply]
@dp20bp Silahkan merujuk pada #166 ditabel ada SLA.
